client/src/ui/Form.jsx

```diff
@@ -326,6 +351,103 @@ export default class Form extends Component {
     }
   }
 
+  getGradeOptions() {
+      const gradeOptions = [];
+      this.gradeRecieved = ["A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", "F"];
```

Comment:
nit: minor typo in variable name
```suggestion
      this.gradeReceived = ["A+", "A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "D-", "F"];
```